app.go

```diff
@@ -39,6 +39,7 @@ func main() {
 	app := cli.App("restorage", "A RESTful storage API with pluggable backends")
 	port := app.IntOpt("port", 8080, "Port to listen on")
 	idMap := app.StringOpt("id-map", "test1:uuid,test2:id,...", "Mapping of collection name to identifier property name")
+	isBinaryUUID := app.BoolOpt("binary-uuid", false, "Is the configured id in a binary format?")
```

Comment:
Also, this option should only apply to "mongo" mode, not "elastic" or any other.